site/src/pages/WorkspacePage/WorkspaceTopbar.tsx

```diff
@@ -221,72 +212,238 @@ export const WorkspaceTopbar: FC<WorkspaceProps> = ({
 							title="Schedule settings"
 							css={{ color: "inherit" }}
 						>
-							Deletion on {new Date(workspace.deleting_at!).toLocaleString()}
+							{workspace.deleting_at ? (
+								<>
+									Deletion on {new Date(workspace.deleting_at).toLocaleString()}
+								</>
+							) : (
+								"Deletion soon"
+							)}
 						</Link>
 					</TopbarData>
 				)}
-
-				{quota && quota.budget > 0 && (
-					<TopbarData>
-						<TopbarIcon>
-							<Tooltip title="Daily usage">
-								<MonetizationOnOutlined aria-label="Daily usage" />
-							</Tooltip>
-						</TopbarIcon>
-						<span>
-							{workspace.latest_build.daily_cost}{" "}
-							<span css={{ color: theme.palette.text.secondary }}>
-								credits of
-							</span>{" "}
-							{quota.budget}
-						</span>
-					</TopbarData>
-				)}
 			</div>
 
-			<div
-				css={{
-					marginLeft: "auto",
-					display: "flex",
-					alignItems: "center",
-					gap: 12,
-				}}
-			>
-				{!isImmutable && (
-					<>
-						<WorkspaceNotifications
-							workspace={workspace}
-							template={template}
-							latestVersion={latestVersion}
-							permissions={permissions}
-							onRestartWorkspace={handleRestart}
-							onUpdateWorkspace={handleUpdate}
-							onActivateWorkspace={handleDormantActivate}
-						/>
-						<WorkspaceStatusBadge workspace={workspace} />
-						<WorkspaceActions
-							workspace={workspace}
-							handleStart={handleStart}
-							handleStop={handleStop}
-							handleRestart={handleRestart}
-							handleDelete={handleDelete}
-							handleUpdate={handleUpdate}
-							handleCancel={handleCancel}
-							handleSettings={handleSettings}
-							handleRetry={handleRetry}
-							handleDebug={handleDebug}
-							handleChangeVersion={handleChangeVersion}
-							handleDormantActivate={handleDormantActivate}
-							handleToggleFavorite={handleToggleFavorite}
-							canDebug={canDebugMode}
-							canChangeVersions={canChangeVersions}
-							isUpdating={isUpdating}
-							isRestarting={isRestarting}
-							isOwner={isOwner}
-						/>
-					</>
-				)}
-			</div>
+			{!isImmutable && (
```

Comment:
All that changed was that, with `WorkspaceScheduleControls` moving to the right of the screen, I can now have everything grouped under a single check of the `immutable` variable

So the parent <div> got moved inside, too, since, even if it doesn't render, the user won't notice